packages/form/src/assets/css/password.css

```diff
@@ -0,0 +1,11 @@
+.eye-icon {
```

Comment:
should be scoped for password input
```suggestion
.input-field-password .eye-icon {
```